Classes/GZNumbers.m

```diff
@@ -0,0 +1,81 @@
+//
+//  GZNumbers.m
+//  Redbooth
+//
+//  Created by Maria Bernis on 20/02/15.
+//  Copyright (c) 2015 teambox. All rights reserved.
+//
+
+#import "GZNumbers.h"
+
+@implementation GZNumbers
+
++ (NSNumber *)randomBOOL
+{
+    return @(arc4random_uniform(2));
+}
+
++ (NSNumber *)randomInteger
+{
+    return @(arc4random_uniform(INT32_MAX));
+}
+
++ (NSNumber *)randomNonZeroInteger
+{
+    return @(arc4random_uniform(INT32_MAX) + 1);
```

Comment:
`return @(arc4random_uniform(INT32_MAX-1) + 1);` or you will cause an overflow :P